### PR TITLE
Ria 2536 record app ho notif

### PR DESCRIPTION
**Pull request by chrisdavecm**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-2536

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/resources/scenarios/RIA-1674-RIA-2936-record-refused-application-ho-and-legal-rep-notifications-after-listing.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
           "applicationDecisionReason": "Some decision reason",
           "applicationSupplier": "The respondent",
           "applicationDecision": "Refused",
-          "notificationsSent": []
+          "notificationsSent": [],
```
Comment (iguannago) [inline src/functionalTest/resources/scenarios/RIA-1674-RIA-2936-record-refused-application-ho-and-legal-rep-notifications-after-listing.json:19]: this has to be outside the replacements field. It's misleading this way.

File: src/functionalTest/resources/scenarios/RIA-1674-RIA-2936-record-refused-application-taylor-house-ho-and-legal-rep-notifications-after-listing.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
           "applicationDecisionReason": "Some decision reason",
           "applicationSupplier": "The respondent",
           "applicationDecision": "Refused",
-          "notificationsSent": []
+          "notificationsSent": [],
```
Comment (iguannago) [inline src/functionalTest/resources/scenarios/RIA-1674-RIA-2936-record-refused-application-taylor-house-ho-and-legal-rep-notifications-after-listing.json:19]: same here

File: src/functionalTest/resources/scenarios/RIA-2536-adjourn-record-refused-application-taylor-ho-and-legal-rep-notifications-prepare-for-hearing.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "ariaListingReference": "LP/12345/2019",
+          "listCaseHearingCentre": "taylorHouse",
+          "applicationType": "Adjourn",
+          "applicationDecisionReason": "Some decision reason",
+          "applicationSupplier": "The respondent",
+          "applicationDecision": "Refused",
+          "notificationsSent": [],
```
Comment (iguannago) [inline src/functionalTest/resources/scenarios/RIA-2536-adjourn-record-refused-application-taylor-ho-and-legal-rep-notifications-prepare-for-hearing.json:19]: and here

**Review by iguannago — DISMISSED** [review]

**Review by Srihari-Nakkala — DISMISSED** [review]
LGTM

**Review by bradsorour — APPROVED** [review]
LGTM

**Review by bradsorour — APPROVED** [review]
LGTM

**Pull request merged**